src/server/status_server/mod.rs

```diff
@@ -39,6 +40,10 @@ use openssl::{
     x509::X509,
 };
 use pin_project::pin_project;
+pub use profile::{
+    activate_heap_profile, deactivate_heap_profile, jeprof_heap_profile, list_heap_profiles,
+    read_file, start_one_cpu_profile, start_one_heap_profile,
```

Comment:
Be cautious with `read_file`. Better not use it.

Reply:
TiFlash implements its own status_server by reusing code from TiKV and adding some specific path to TiKV's status_server.
We expose `read_file` because it is used in TiKV's status_server